src/components/chip/chip.tsx

```diff
@@ -251,13 +259,19 @@ export class Chip implements ChipInterface {
         return <span class="text">{this.text}</span>;
     };
 
-    private renderIcon() {
+    private renderPicture() {
         const icon = getIconName(this.icon);
 
-        if (!icon) {
+        if (!icon && !this.image) {
             return;
         }
 
+        if (this.image) {
+            return (
+                <img src={this.image.src} alt={this.image.alt} loading="lazy" />
```

Comment:
Based on what I understand from the cryptic descriptions on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#referrerpolicy), it's better/safer if we add `referrerpolicy="no-referrer"` to the image.
```tsx
<img src={this.image.src} alt={this.image.alt} loading="lazy"  referrerpolicy="no-referrer"/>
```

However, adding it like that in the code will show a ts error:
```
Type '{ src: string; alt: string; loading: "lazy"; referrerpolicy: string; }' is not assignable to type 'ImgHTMLAttributes<HTMLImageElement>'.
  Property 'referrerpolicy' does not exist on type 'ImgHTMLAttributes<HTMLImageElement>'.ts(2322)
```

Reply:
The `img` interface doesn't have a property called `referrerpolicy`. It doesn't seem we can implement that. 

```
   interface ImgHTMLAttributes<T> extends HTMLAttributes<T> {
        alt?: string;
        decoding?: 'async' | 'auto' | 'sync';
        importance?: 'low' | 'auto' | 'high';
        height?: number | string;
        loading?: 'lazy' | 'auto' | 'eager';
        sizes?: string;
        src?: string;
        srcSet?: string;
        srcset?: string;
        useMap?: string;
        usemap?: string;
        width?: number | string;
    }